third_party/rocm/LICENSE

```diff
@@ -0,0 +1,27 @@
+Copyright (c) 2018 Advanced Micro Devices, Inc. All Rights Reserved.
```

Comment:
@ThomasRaoux You may want to check the import scripts on the Google side: we don't want this being pulled into our internal repo in this form.

Reply:
third_party is not being merged on the Google side and since this is experimental it won't build by default so it should be fine to merge as is. 
That being said I'll follow up with build cop to make sure everything goes smoothly.